app/views/packages/show.html.erb

```diff
@@ -1,2 +1,6 @@
 <h1>Packages#show</h1>
 <p>Find me in app/views/packages/show.html.erb</p>
+
+<% @photographers_present.each do |pgr| %>
```

Comment:
@Matth93 just make sure you remove this until we decide how we'll present the photographer cards :)